react/components/Prefetch/PQueue.ts

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable */
```

Comment:
why disable eslint?

Reply:
it was complaining about the empty methods